apps/dg/components/case_table/case_table_view.js

```diff
@@ -867,6 +867,14 @@ DG.CaseTableView = SC.View.extend( (function() // closure
                                       if( !ioMenuItem.disabled)
                                         ++enabledItems;
                                    });
+          // add 'dg-wants-touch' class to menus
+          setTimeout(function() {
+            $('.slick-header-menu')
+              .addClass(function(index, classes) {
+                return (!classes || (classes.indexOf('dg-wants-touch') < 0))
+                        ? 'dg-wants-touch' : '';
+              });
```

Comment:
I think that this code is adding 'dg-wants-touch' if not already present. I think this is already the behavior of $(..).addClass(string). Is there more to this that I am missing?

Reply:
That may be the behavior of `addClass`, but it's not documented as such. The closest the docs get to addressing the question is "It's important to note that this method does not replace a class. It simply adds the class, appending it to any which may already be assigned to the elements." -- which sounds more like appending the class independent of whether it already exists or not.

Reply:
I agree that its not documented as such, so your implementation makes sense. It would be possible that they could change the behavior and still be true to the documentation. I stand corrected.